internal/tenancy/tenancytest/namespace_test.go

```diff
@@ -5,11 +5,12 @@ package tenancytest
 
 import (
 	"context"
+	"testing"
+
 	"github.com/hashicorp/consul/agent/grpc-external/services/resource"
 	svctest "github.com/hashicorp/consul/agent/grpc-external/services/resource/testing"
-	resource2 "github.com/hashicorp/consul/internal/resource"
+	resourceTenancy "github.com/hashicorp/consul/internal/resource"
```

Comment:
The reason I suggested `svc` for L10 is because it aligns with `svctest` on L11 (the packages are the same except the latter has "testing".

L12 should not need an alias because the rest of the code base is pretty consistent about that: https://github.com/search?q=repo%3Ahashicorp%2Fconsul++%2F%22github.com%5C%2Fhashicorp%5C%2Fconsul%5C%2Finternal%5C%2Fresource%22%24%2F&type=code

Reply:
Your suggestion did not load in time 😵‍💫